crates/evm/core/src/fork/backend.rs

```diff
@@ -202,11 +202,11 @@ where
     fn get_account_req(&self, address: Address) -> ProviderRequest<eyre::Report> {
         trace!(target: "backendhandler", "preparing account request, address={:?}", address);
         let provider = self.provider.clone();
-        let block_id = self.block_id;
+        let block_id = self.block_id.unwrap_or(BlockId::latest());
```

Comment:
I think you could use `unwrap_or_default()` here as it will use `BlockId::latest()` by default